crates/invoker-impl/src/invocation_task.rs

```diff
@@ -301,7 +299,7 @@ where
     }
 
     /// Loop opening the request to deployment and consuming the stream
-    #[instrument(level = "debug", name = "invoker_invocation_task", fields(rpc.system = "restate", rpc.service = %self.service_id.service_name, restate.invocation.id = %self.invocation_id), skip_all)]
+    #[instrument(level = "debug", name = "invoker_invocation_task", fields(rpc.system = "restate", rpc.service = %self.invocation_target.service_name(), restate.invocation.id = %self.invocation_id, restate.invocation.target = %self.invocation_target), skip_all)]
```

Comment:
Wouldn't `self.invocation_target.service_name()` be contained in `restate.invocation.target = %self.invocation_target`?

Reply:
yes, but the fields `rpc.service` and `rpc.method` are standards part of the otel semantic convention spec, while `restate.invocation.target` is something custom, so i would prefer to fill both when possible.